w3d_translator/utils.py

```diff
@@ -1,6 +1,13 @@
 from errors import TranslationError
 
 
+def split_str(string: str, parenthesis=True) -> list:
```

Comment:
it doesn't look like the optional parenthesis arg is used

Reply:
Oh yes that's old, will delete!